packages/keymaster/src/db-wallet-json-enc.js

```diff
@@ -0,0 +1,84 @@
+import fs from 'fs';
+import crypto from 'crypto';
+
+const dataFolder = 'data';
+const walletName = `${dataFolder}/wallet.json`;
```

Comment:
Should the encrypted wallet use a different filename? Since it overwrote data/wallet.json it currently breaks keymaster-api and satoshi-mediator.

Reply:
I was originally going to do that, but wallets tend to encrypt the existing wallet. Happy to use a different filename rather than break things, it will then be up to the user to manage wallet files. Let me know if you want me to use `wallet-enc.json` or something similar for the encrypted wallet.

Reply:
Let's use `wallet-enc.json` for now, then when we upgrade the other dependent services we can change it to encrypt in place later.